src/utils/lorem-utils.js

```
@@ -0,0 +1,211 @@
+    const sentences = [];
+    for (let i = 0; i < count; i++) {
+      const wordCount = this.randomBetween(8, 20);
+      const words = this.generateWords(wordCount).split(' ');
+
+      words[0] = this.capitalizeFirst(words[0]);
+
+      let sentence = words.join(' ');
+
+      if (Math.random() < 0.1) {
+        sentence += '!';
+      } else if (Math.random() < 0.1) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)